public_html/wp-content/mu-plugins/blocks/assets/src/blocks.js

```diff
@@ -6,10 +6,14 @@ const { registerBlockType } = wp.blocks;
 /**
  * Internal dependencies
  */
+import * as sessions from './sessions/';
 import * as speakers from './speakers/';
+import * as sponsors from './sponsors/';
 
 [
+	sessions,
 	speakers,
+	sponsors,
```

Comment:
`MIGHT` - Ideally, it seems like these lines should be added in 20dce6a and 6bf35ce. If that results in a bunch of merge conflicts, though, then it may not be worth the time in this case. If it only takes a few minutes, though, then I think it'd be worth it.